strapi/strapi
# Add nvm config file

### What does it do?

Add config file for [NVM](https://github.com/nvm-sh/nvm/).

### Why is it needed?

- Having a `.nvmrc` file makes it helpful to dictate which node version is currently used for project development.

### How to test it?

- In the project root dir run `nvm install && nvm use`. 
- Check your node version by running `node -v`.
- Your current node version should be the same as defined in `.nvmrc` file.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
